combat-trainer.lic

```diff
@@ -44,6 +50,9 @@ class SetupProcess
     was_retreating = game_state.retreating?
     game_state.update_room_npcs
     if game_state.dancing?
+      if @use_warhorn
```

Comment:
This could be done cleaner two ways. 

```
blow_warhorn if @use_warhorn
```
or
```blow_warhorn``` and in the method
```
def blow_warhorn()
  return unless @blow_warhorn
  return if Time.now - @last_warhorn < 300
```

The second one follows the norm for the rest of Dependency.

Reply:
Sounds good. But do you mean:
return unless @warhorn

to be consistent with dropping a flag yaml setting